src/REstate/Engine/TransitionNotDefinedException.cs

```diff
@@ -2,36 +2,22 @@
 
 namespace REstate.Engine
 {
-    public class TransitionNotDefinedException<TState, TInput>
```

Comment:
Changing this to non-generic since this needs to move across network boundaries, we need to keep exceptions minimal.